getting_started.md

```diff
@@ -0,0 +1,168 @@
+For now, we're using Google Cloud Platform (GCP), since projects like Terra
+and Hail so far work best on GCP.
+
+If you're new to GCP, read the excellent [How to Cloud] guide first.
+
+[How to Cloud]: https://github.com/danking/hail-cloud-docs/blob/master/how-to-cloud.md
+
+For better isolation and cost accounting, we create separate GCP projects for
+each effort within the Centre. For example, the TOB-WGS effort would have a
+dedicated GCP project called `tob-wgs`. It's important to keep in mind that
+the GCP project *name* can be distinct from the project *ID*. In general,
+when you specify projects, you'll have to use the project ID (e.g. `gcloud
+config set project <project-id>`).
+
+Permissions to projects and resources like Google Cloud Storage (GCS) buckets
+are managed using Google Groups that are linked to IAM permission roles. Take
+a look at our [storage policies] for a much more detailed description.
+
+[storage policies]: https://github.com/populationgenomics/storage-policies
+
+Please avoid storing any non-public data on your laptop, as that increases the
+risk of accidental data breaches significantly. Keep in mind that any
+non-public genomic data is highly sensitive.
+
+# Hail
+
+[Hail] is an amazing open source library and platform for genomic data
+analysis, developed at the Broad Institute. Given it's proven scalability and
+our good relationship with the Hail development team, it's the Centre's main
+analysis platform.
+
+[Hail]: https://hail.is
+
+TODO: installation (Through CPG Bioconda channel?)
+
+The Hail [documentation] is a good starting point. In particular, the
+[tutorials] are worth looking into.
+
+[documentation]: https://hail.is/docs/0.2/index.html
+[tutorials]: https://hail.is/docs/0.2/tutorials-landing.html
+
+Don't feel discouraged if the "lazy evaluation" model of Hail feels
+unintuitive at first. It takes some time getting used to, but it's extremely
+powerful.
+
+There's also a [workshop recording] that contains a lot of useful tips, although
+not everything is applicable to the Centre.
+
+[workshop recording]: https://drive.google.com/file/d/1c5us8YSApSGl81CrojeR426wTS2QA53d/view?usp=sharing
+
+If you have any Hail related questions, feel free to ask on Slack in the
+`#team-analysis` channel. The Hail team is also very responsive in the
+[Zulip chat], but you'll have to take the time zone difference into account.
+Finally, there's also an official [discussion forum].
+
+[Zulip chat]: https://hail.zulipchat.com
+[discussion forum]: https://discuss.hail.is/
+
+At the moment, using Hail requires launching a Dataproc (Spark) cluster.
+Please always set a maximum age for the cluster (`--max-age` below), to avoid
+accidental spending in case you forget to stop the cluster after your job has
+completed:
+
+```bash
+hailctl dataproc start --max-age 2h --region australia-southeast1 my-cluster
+```
+
+If you're interested in the Hail internals, this developer focussed
```

Comment:
That really is an awesome overview, was reading it earlier today. Really helps in mapping out the codebase.